From 9fa997dbc9098ae5184bd5359ec4354900ae121b Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:32:32 +0800
Subject: [PATCH 1/3] clk: aspeed: add mac.o for AST2705

Wire CONFIG_ASPEED_AST2705 into the clock Makefile so the MAC clock
driver is built for AST2705 boards.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Icd6014849f8570387aa28cf33be5d4d3d79c3b85
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 drivers/clk/aspeed/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/aspeed/Makefile b/drivers/clk/aspeed/Makefile
index 0041970d88..3504231e6c 100644
--- a/drivers/clk/aspeed/Makefile
+++ b/drivers/clk/aspeed/Makefile
@@ -6,4 +6,5 @@
 obj-$(CONFIG_ASPEED_AST2500) += clk_ast2500.o
 obj-$(CONFIG_ASPEED_AST2600) += clk_ast2600.o
 obj-$(CONFIG_ASPEED_AST2700) += mac.o
+obj-$(CONFIG_ASPEED_AST2705) += mac.o
 obj-$(CONFIG_CLK_AST2700) += clk_ast2700.o
-- 
2.43.0

